### DISCUSSION TITLE
World scale slider

### DISCUSSION THREAD

**Discussion by hundenmarvin** [fholger/crysis_vrmod#13] [Ideas]
Before this mod switched from OpenVR to OpenXR you could change the scale of the world in the SteamVR settings. It was really nice to play it at 50 % scale. Everything was so much closer and gave you this "3D pop" sensation. And if you played in 3rd person while driving a car you could set the scale to 10 %. It was like steering an RC car on a toy sized island. It was incredible! It would be nice if you implemented a world scale slider in the mod now that it is unavailable because of OpenXR.

**Comment by Braxusx** [comment]
I think he switched to OpenXR for compatibilities sake. Plus everything is moving to OpenXR anyway. I'm just glad he's working on it and I hope this project sees it all the way through. It's a great game and we need cool modders like fholger.

Just curious, what HMD are you using and on what PC specs?

I've tried it with Reverb G2 and Quest 2 on a 3090 + 10900k + 32gb ram.

**Comment by hundenmarvin** [comment]
Yeah, I agree. OpenXR is great. It might mean we can utilize the superior ASW2. And the possible implementation of room scale and motion controls would be way way better than just a world scale slider.

I have an Index and a Quest 2. I have only tried it on Index because Quest 2 is to heavy for my neck if I am not moving around. My computer specs are i7-6700K, 16gb, 1080ti. I am playing on high graphics settings. The framerate doesn´t seem to be great but I am pretty used to it. I have been planning to buy a better computer but the prices never seem to go down.

**Comment by xPawelx** [comment]
Hi all. 
I've tried it on i7 13700k and rtx4080 with quest 2 at resolution 4056x4104 per eye. It works great and a clarity of picture is like playing 2D at resolution of 1920x1080. I thing world scale option would be great. I find it very useful in FS2020. Also I can't wait for controller to be working like in Black Mesa VR. It allows to keep your rifle in two hands... great feeling....
